src/prefect/settings.py

```diff
@@ -312,6 +312,25 @@ def max_log_size_smaller_than_batch_size(values):
     return values
 
 
+def status_codes_as_integers_in_range(value):
+    """
+    Validator for settings asserting the status codes are integers in the range 100-599
+    """
+    if value == "":
+        return set()
+
+    values = {v.strip() for v in value.split(",")}
+
+    if any(not v.isdigit() or int(v) < 100 or int(v) > 599 for v in values):
+        raise ValueError(
+            "PREFECT_SERVER_HEALTHY_STATUS_CODES must be a comma separated list of "
```

Comment:
Looks like this refers to the wrong setting

Reply:
Whoops... yep, need to clean up the name.